src/helpers/Monitor.hpp

```diff
@@ -11,7 +11,16 @@
 #include "Region.hpp"
 #include <optional>
 
+// Enum for the different types of auto directions, e.g. auto-left, auto-up.
+enum class AutoDirs {
```

Comment:
enums prefix `e`: `eAutoDirs`

Reply:
I see all of the other classes and structs are prefixed with a capital letter. e.g. SMonitorRule. Is that prefered over lower case? I'll stick with what you said here for now.